spec/services/academic_record_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+describe AcademicRecord do
+  it 'Saves the new student in the database' do
+    new_student = build(:student)
+    subjects = [build(:subject), build(:subject)]
+
+    subject.enroll_student(new_student, subjects)
```

Comment:
`expect { subject.enroll_student(new_student, subjects) }.to change { Student.count }.by 1`